src/devtools/client/inspector/markup/actions/markup.ts

```diff
@@ -133,15 +134,21 @@ export function setupMarkup(store: UIStore, startAppListening: AppStartListening
         }
       }
 
-      // If the "Elements" panel is already selected, go ahead and fetch markup data now.
-      // Otherwise, wait until the next time it _is_ selected.
-      if (isInspectorSelected(getState())) {
-        await loadNewDocument();
-      } else {
+      if (!isInspectorSelected(getState())) {
         await condition((action, currState) => {
           return isInspectorSelected(currState);
         });
+      }
+
+      try {
         await loadNewDocument();
+      } catch (error) {
+        console.error(error);
```

Comment:
Should we try to narrow the error down to avoid catching other errors? I'm not sure how many types of error could be thrown here.